### DISCUSSION TITLE
Output without hooks, only types

### DISCUSSION THREAD

**Discussion by jokull** [orval-labs/orval#378] [Ideas]
I have backend and multiple frontend repos. I want to extract openapi docs into a separate repo and have each project generate their own types. 

For the backend however, I don't need the hooks, only the types. Would it make sense to allow this config to drop hooks completely?

```js
module.exports = {
  petstore: {
    output: {
      client: null,
    },
  },
};
```

**Comment by anymaniax** [comment]
Hi if you don't put a client and just add a path to the property schemas it should work as you expect normally.

**Reply by xopher-b** [reply]
Does this still work?

**Comment by kostia1st** [comment]
I upvote this. 

I also want to opt out of client generation completely.

There's that `client` prop having `axios-functions` [by default](https://orval.dev/reference/configuration/output#client). There seems to be no way to set `null` or `undefined` instead.

**Comment by velizcode** [comment]
I have the following configuration:

```
// orval.config.ts
import { defineConfig } from 'orval';

export default defineConfig({
  'x': {
    output: { schemas: './src/api/models' },
    input: '../swagger-spec.json',
  },
});

```
this conf generates only types

**Reply by jdmaldonado** [reply]
Is there any solution? even the current version 8 is forcing to have target which is generating the client.
